docs/changes/2736.feature.rst

```diff
@@ -0,0 +1,7 @@
+The muon ring fitter needs to be improved.
```

Comment:
As I wrote before, this is a changelog entry. It describes what has already happened.

Reply:
This comment was not addressed. Please leave it to the maintainers to resolve conversations.

Please change this sentence to something like: "The algorithms for muon ring fitting have been improved:"